starfish/core/types/_spot_finding_results.py

```diff
@@ -0,0 +1,86 @@
+from typing import List, Mapping, MutableMapping, Optional, Tuple
+
+from starfish.core.types import Axes, SpotAttributes
+
+AXES_ORDER = (Axes.ROUND, Axes.CH)
+
+
+class SpotFindingResults:
+    """
+    Wrapper class that describes the results from a spot finding method. The results
+    mapping is a collection of (round, ch) indices and their corresponding measured
+    SpotAttributes.
+    """
+
+    def __init__(self, spot_attributes_list: Optional[List[Tuple]] = None):
+        """
+        Construct a SpotFindingResults instance
+
+        Parameters
+        -----------
+        spot_attributes_list : Optional[List[Tuple]]
+            If spots were found using ImageStack.transform() the result is a list of
+            tuples (indices, SpotAttributes). Instantiating SpotFindingResults with
```

Comment:
The format for the indices is not very clear.  It appears to be a Tuple of index values in the order of AXES_ORDER, but I'm just inferring that from the code.

Reply:
That's exactly what it is! I'll make it more clear